### DISCUSSION TITLE
Protected Message(s)

### DISCUSSION THREAD

**Discussion by seanollett** [community/community#128105] [New to GitHub]
Hi, I am new to github. This morning I received an email telling me I have rceived a protected message wgich I can access at : https://webm-cpl4.github.io/?QZFbHlFbOBTYxYVNNZkVsZlbShUWWh2SSFDZUNWRadlV6ZkVUxmSDVWbNp3UqpkTkFDczdFWONkYrRTP=&y=AUUFzMUhEbw0UbRhnYE5kWNpHbUdVbzFTUwUTNRZFaoRGRShlWXNXMlZFbwdFbSpVYExWNURkWON1RNdXVrhGa&e=c2VhbkBvbGxldHQtZmFtaWx5LmNvbQ
Is this genuine because I have never received one of these before and don't want any problems. Thanks for your help in advance. Sean

**Comment by VargaFelixErvin** [comment]
Who sent the email?
Does the email come from a real @github.com domain?

A “protected message” email often means someone sent you an encrypted or secure message through a third-party service. Common in:
Phishing scams
Corporate email systems (e.g., from Microsoft 365) if someone shares a protected doc
But GitHub doesn't use this format for messages.
